src/main/java/com/minecolonies/network/messages/SaveScanMessage.java

```diff
@@ -0,0 +1,85 @@
+package com.minecolonies.network.messages;
+
+import com.minecolonies.util.LanguageHandler;
+import com.minecolonies.util.StructureWrapper;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.client.Minecraft;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.client.Minecraft'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)